aderyn_core/src/detect/low/reverts_and_requries_in_loops.rs

```diff
@@ -71,7 +71,7 @@ mod reevrts_and_requires_in_loops {
     };
 
     #[test]
-    fn test_reverts_and_requires_in_loops() {
+    fn test_reverts_and_requires_in_loops_by_loading_contract_directly() {
```

Comment:
Detector tests must contain `by_loading_contract_directly` otherwise it is susceptible to fail in CI due to `ExecutableFileBusyError`.